core.md

```diff
@@ -0,0 +1,127 @@
+
+## Definitions
+
+Following are some core definitions that will be referred to in the later sections of the document.
+
+### The Project
+
+"The Project" refers to The NeoForged project, with a presence on both GitHub and Discord, as noted at the start of this document. It does not refer to any specific repository or subproject.
+
+### Subprojects
+
+Subprojects are large tasks that the Project itself is undergoing.
+
+These may be:
+
+* Repositories under the Project's GitHub.
+* Large, multi-repository tasks that are being undertaken (ie. updating the API to a new version of the Game).
+* Smaller, more targetted tasks that are being undertaken (ie. refactoring the Modloader or API to remove or replace or rework a feature).
+
+### Teams
+
+The Project is split into four teams.
+
+A team is a group of people with similar goals, and equal weighting in most decision making - with some exceptions, which will be explored later.
+
+You may [apply to join some teams](https://links.neoforged.net/apply), but some require appointment from within an existing team, or further voting after application.
+
+#### Steering Council
+
+The Steering Council is the overall management and oversight of the Project. 
+
+They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+
+The Council has at all times an odd number of active members, such that stalemates are not possible.
+
+A member may choose to become inactive and appoint another in their place until they return.
+
+At least one member of the Council is promoted from other teams via an annual internal vote, to ensure that the Council always has fresh insight from a new team member with a different view on situations, and thus does not stagnate over time.
+
+Once a year, [an internal voting process](https://peps.python.org/pep-8104/) will determine the members that will be promoted to the Steering Council, via the ranked voting system. It is intended that at least one, and at most three members will be promoted at a time.
+
+The Council also serves double duty as a mediator for disagreements between other teams. These shall be resolved by discussion, rather than by unanimous applications of the rules, so this is not in the purview of this document.
+
+However, the Council does not have all-encompassing powers. They are the avenue of last resort, and the vast majority of decisions should be made by the team in charge, not by the Steering Council.
+
+The Council has these powers to resolve the 1% of exceptional situations, not the 99%.
+
+#### Maintainers
+
+The Maintainers team consists of all project developers who have push access or technical rights to the project's repositories or infrastructure.
```

Comment:
What is "technical rights"?

Reply:
This one was taken from @sciwhiz12's draft, you'd have to ask him.

Reply:
Technical rights/access means the user has access to technical configuration or operations. It can take many forms, according to their context:
- For GitHub repositories, it chiefly refers to Write permission and above; having the permission to push commits to branches.
- For services, it refers to being able to reconfigure, enable, or disable service components.
- For server access, it refers to the access to login and access a console or configuration page for a server.

It is qualified as "technical" to distinguish it from other forms of rights/access, such as moderation rights/access (the ability to moderate users and their contents).